src/domain/session/room/RoomViewModel.js

```diff
@@ -197,18 +197,80 @@ export class RoomViewModel extends ViewModel {
         }
     }
     
+    async _getMessageInformations (message) {
+        let msgtype = "m.text";
+        if (message.startsWith("/")) {
+            let command=message.substring(1).split(" ");
+            switch (command[0]) {
+                case "me":
+                   message = message.substring(4).trim();
+                   msgtype = "m.emote";
+                   break;
+               case "join":
+                   if (command.length==2) {
+                       let roomname = message.substring(5).trim();
+                       try {
+                           await this._options.client.session.joinRoom(roomname);
+                       } catch (exc) {
+                           if (exc.statusCode??exc.status==400) {
```

Comment:
Also you probably meant to do `===` instead of `==`?

Reply:
Why do we need to increase the verification level?

Reply:
`===` is strict equality check, see https://stackoverflow.com/a/359509

Reply:
Yes I know... But if the status is `"404"` instead of being `404` (and I am not sure that it is possible) what's the problem?

Reply:
It's a code quality thing i.e if you have the choice of choosing between `===` or `==`, you should always choose `===`.

Reply:
Ok, thanks

Reply:
Basically, we always use === unless there is a good reason to use == (which hasn't happened yet I think?), so we are less likely to get it wrong when it does matter :)